docs/product-user-guide/src/main/asciidoc/data-objects-create-proc.adoc

```diff
@@ -1,30 +1,30 @@
-[id='data_objects_create_proc']
-= Creating a Data Object
+[id='data-objects-create-proc']
+= Creating data objects
 
-The data objects that you define can be implemented in various types of assets in your project, and are essential building blocks for your project.
+The data objects that you define can be implemented in various types of project assets, such as in guided decision tables and guided rules, and are the building blocks for the assets in your project.
 
 .Procedure
 . Go to *Menu* -> *Design* -> *Projects* and click the project name.
 . Click *Create New Asset* → *Data Object*.
-. Enter the name and select the package. The name must be unique across the package, but it is possible to have two data objects with the same name in two different packages.
-. To make your Data Object persistable, check the *Persistable* checkbox.
-. Click *Ok*.
-. Create fields of the data object:
+. Enter a unique *Data Object* name and select the *Package* where you want the data object to be available for other rule assets. Data objects with the same name cannot exist in the same package specified. The package that you specify must be the same package where the rule assets that require those data objects have been assigned or will be assigned.
 +
-.. Click *add field* in the main editor window to add a field to the object with the attributes *Id*, *Label* and *Type*. Required attributes are marked with *.
-* *Id*: The ID of the field unique within the data object.
-* *Label*: The label to be used in the *Fields* panel. This field is optional.
-* *Type*: The data type of the field.
+.Using data objects across packages
+[IMPORTANT]
+====
+Data objects and the assets that require those data objects must be in the same package. Even if the asset and data object are in the same project, they cannot be used together unless they are placed in the same package. You can either define the package as you create new data objects, as described in this section, or import an existing data object into the package of another asset by using the *Data Objects* tab of the specific rule asset designer. For example, see xref:guided-decision-tables-columns-create-proc[].
```

Comment:
IMO this wording is confusing. You start by saying data objects must be in the same package and then state they can be imported. It might be better to re-phrase as something along the lines of "Rule assets only have visibility of Data Objects in the same Package unless they have been imported using the Data Objects tab".

Reply:
Yeah, I see what you mean. Since the fact that the data objects need to be in the same package, etc., is already noted in the actual step, I've just changed this section to focus on importing, as an alternative. Clearer now, I think: http://file.rdu.redhat.com/~sterobin/BXMSDOC-2008/#data-objects-create-proc